themes/jquery/js/sri-modal.js

```diff
@@ -4,12 +4,17 @@ $( function() {
 
 	// Show modal on click
 	$( "body" ).on( "click", ".open-sri-modal", function( event ) {
+		var href, link;
+
 		if ( event.ctrlKey || event.metaKey ) {
 			return;
 		}
 
+		href = $( this ).attr( "href" );
+		link = isRelativeUrl( href ) ? document.location.origin + href : href;
```

Comment:
I could be wrong, but would `this.href` make this code redundant? Looks like we just want to expand it to its natural absolute url target, right? Property access does that natively.

Reply:
Good catch; I should just change `attr` to `prop` here.

Reply:
Updated.